### DISCUSSION TITLE
Cloudflare workers for ublock code execution

### DISCUSSION THREAD

**Discussion by Inglan2** [3kh0/ext-remover#1108] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I know ublock is being deperacted next year because of mv3.

But still...

So im thinking of using cloudflare workers to execute script dynamically because eval doesnt work everywhere with ublock. 

Bascially you write script and get a url that you put into your ublock and then you can execute script!

Ill work on it soon

**Comment by nathblade16** [comment]
this is a really cool concept and I would love to see it exist, I don't really know what a cloudflare worker is though so can you please explain it?

**Reply by Inglan2** [reply]
A cloudflare worker I think is a serverless function. So whenever the url is accesed it executes code and can return stuff. So good for making apis in some cases

**Reply by nathblade16** [reply]
> A cloudflare worker I think is a serverless function. So whenever the url is accesed it executes code and can return stuff. So good for making apis in some cases

this does sound really cool if you can make it work... maybe test it and see if it works

**Comment by Blobby-Boi** [comment]
Code execution doesnt work everywhere since csp policy and stuff so how would this bypass it

**Reply by Inglan2** [reply]
instead of using `eval()` in a script it just creates a new script basically

**Comment by Inglan2** [comment]
Workers.dev is blocked at our school so I'll try and get an eu.org sub as they are unblocked
